.rubocop_todo.yml

```diff
@@ -0,0 +1,841 @@
+# This configuration was generated by
```

Comment:
The rubocop_todo should inherit .rubocop.yml, see comment on .rubocop.yml for further explanations.